### PR TITLE
🐛 Fix dependency install bug

### PR DESCRIPTION
**Pull request by SimmonsRitchie**
## Summary
Updates our Pipfile to fix a dependency conflict issue that has caused our scheduled Github action [to fail](https://github.com/City-Bureau/city-scrapers-akr/actions/workflows/cron.yml) for the past two months. It also:

- Requires Python 3.9 in our Pipfile
- Specifies Python 3.9 in our Github workflows
- Re-formats five files based on the style rules of a newer version of Black
- Removes three packages that appear to be unused (chardet, twisted and pypiwin32)

## Why are we doing this?
We want working scrapers! These changes are modeled heavily on the Pipfiles of our others repos, particularly [city-scrapers](https://github.com/City-Bureau/city-scrapers), which are functioning properly.

These changes appear to have achieved their goal: both our CI and cronjob workflows are now [working](https://github.com/City-Bureau/city-scrapers-akr/actions/runs/7281753095).

## How can these changes be manually tested?

1. If you've previously installed this project, remove your existing virtual environment:
`pipenv --rm`
2. Reinstall the project and ensure that there are no errors:
`pipenv sync --dev`
3. Run a few of our checks to simulate the checks we run in our CI process:
`pytest && black . --check`
4. Ensure our custom Scrapy command is working properly and doesn't throw any errors:
`scrapy validate`

For extra credit, you can check that the cronjob workflow works as expected by manually triggering it from [this page](https://github.com/City-Bureau/city-scrapers-akr/actions/workflows/cron.yml) and using the dropdown menu (pictured below). Just note that it will take about an hour to run:

image

## Are there any smells or added technical debt to note?
- I've specified the Python venv as 3.9 because I think it's ideal for our Github workflows and our dev environment to use the same environment. I've observed a number of dependency conflicts in this and our other city-scraper repos (eg. [city-scrapers](https://github.com/City-Bureau/city-scrapers/pull/1056)) and I think it will make these repos easier to maintain and easier to debug in future. I welcome feedback on this choice.
- pypiwin32 is among one of the packages removed. I believe this package was originally installed in order to help Windows-users set up their dev environment. My understanding is this package is outdated and we don't need it. I have more thoughts and research notes in a separate PR [here](https://github.com/City-Bureau/city-scrapers/pull/1056).

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,19 +4,19 @@ verify_ssl = true
...
 [dev-packages]
 freezegun = "*"
 pytest = "*"
 "flake8" = "*"
 isort = "*"
-black = "==19.10b0"
+black = "*"
+
+[requires]
+python_version = "3.9"
```
Comment (LienDang) [inline Pipfile:22]: @SimmonsRitchie you can configure your IDE to automatically add a new line at the end of file on save, so Github won't show the red error icon above. See more here https://thoughtbot.com/blog/no-newline-at-end-of-file

**Comment by LienDang** [comment]
Looks good to me!

File: Pipfile (hunk 1 above)
Comment (SimmonsRitchie) [reply Pipfile]: Nice catch and great tip! Thanks, @LienDang 😊

**Comment by SimmonsRitchie** [comment]
Excellent! Thanks, @LienDang! Merging now 🚀

**Pull request merged**